src/coreclr/src/debug/createdump/main.cpp

```diff
@@ -40,6 +40,15 @@ int __cdecl main(const int argc, const char* argv[])
     int exitCode = 0;
     int pid = 0;
 
+#ifdef __APPLE__
+    char* enabled = getenv("COMPlus_DbgEnableElfDumpOnMacOS");
+    if (enabled == nullptr || strcmp(enabled, "1") != 0)
+    {
+        fprintf(stderr, "Dumps not supported on MacOS\n");
```

Comment:
Maybe the error message should mention the name of the environment variable that controls creating dumps.

Reply:
Done